evm-adapters/src/sputnik/cheatcodes/cheatcode_handler.rs

```diff
@@ -35,6 +33,9 @@ pub static CHEATCODE_ADDRESS: Lazy<Address> = Lazy::new(|| {
     Address::from_slice(&hex::decode("7109709ECfa91a80626fF3989D68f67F5b1DD12D").unwrap())
 });
 
+pub static CONSOLE_LOG_ADDRESS: Lazy<Address> = Lazy::new(|| {
+    Address::from_slice(&hex::decode("000000000000000000636F6e736F6c652e6c6f67").unwrap())
+});
```

Comment:
New line below this - is this the hardhat console address? Can we add a link to it from the hardhat codebases?